atc/atccmd/command.go

```diff
@@ -380,10 +379,18 @@ func (cmd *Migration) rotateEncryptionKey() error {
 }
 
 func (cmd *Migration) migrateToLatestVersion() error {
+	lockConn, err := constructLockConn(defaultDriverName, cmd.Postgres.ConnectionString())
+	if err != nil {
+		return err
+	}
+	defer lockConn.Close()
+
+	lockFactory := lock.NewLockFactory(lockConn, metric.LogLockAcquired, metric.LogLockReleased)
+
 	helper := migration.NewOpenHelper(
 		defaultDriverName,
 		cmd.Postgres.ConnectionString(),
-		nil,
+		lockFactory,
```

Comment:
Could you pass the lock factory in for *all* the `concourse migrate` commands (at the very least, the ones that mutate the DB - `migrateDBToVersion` and `rotateEncryptionKey` as well)

Reply:
(I recognize that the other commands aren't used in the chart, but it seems like something we should probably have anyway)